README.md

```diff
@@ -85,13 +148,139 @@ sudo crontab -e
 Place the following at the end of the file, then close and save it.
 
 ``` sh
-0 23 * * * docker run --rm --name certbot -v "/dojot/etc/letsencrypt:/etc/letsencrypt" -v "/dojot/var/lib/letsencrypt:/var/lib/letsencrypt" -v "/dojot/data/letsencrypt:/data/letsencrypt" -v "/dojot/var/log/letsencrypt:/var/log/letsencrypt" certbot/certbot renew --webroot -w /data/letsencrypt --quiet && docker restart https-nginx
+0 23 * * * docker run --rm --name certbot -v "/dojot/etc/letsencrypt:/etc/letsencrypt" -v "/dojot/var/lib/letsencrypt:/var/lib/letsencrypt" -v "/dojot/data/letsencrypt:/data/letsencrypt" -v "/dojot/var/log/letsencrypt:/var/log/letsencrypt" certbot/certbot renew --webroot -w /data/letsencrypt --quiet && chmod 0755 /dojot/etc/letsencrypt/{live,archive} && docker restart dojot_apigw_1
 ```
 
-The above command will run every night at 23:00, renewing the certificate and forcing Nginx to restart
-if the certificate is due for renewal.
+The above command will run every night at 23:00, renewing the certificate and forcing Kong to restart if the certificate is due for renewal.
 
-## Disclaimer
+##### Use HTTPS with self-signed certificate
 
-This deployment option is best suited to development and functional environments.
-For production environment we recommend to use Kubernetes.
+As prerequisites this uses [git](https://git-scm.com/) and [OpenSSL](https://www.openssl.org/).
+
+On Debian-based Linux distributions, you can install these prerequisites by running:
+
+``` sh
+sudo apt install git openssl
+```
+
+After installing the prerequisites if necessary, download the tool to generate certificates [certgen](https://github.com/dojot/dojot/tree/development/tools/certgen) (to see other possible parameters go to [documents](https://github.com/dojot/dojot/tree/development/tools/certgen#readme)):
+
+```sh
+git clone --single-branch --branch development https://github.com/dojot/dojot.git cert-tools-temp && cp -r cert-tools-temp/tools/certgen certgen && rm -rf cert-tools-temp
```

Comment:
To test use https://github.com/dojot/dojot/tree/feature/certgen-self-rsa/tools/certgen

Reply:
to test use feature/certgen-self-rsa instead of development  for now